CRM/Core/DAO.php

```diff
@@ -647,6 +647,9 @@ public function copyValues(&$params, $serializeArrays = FALSE) {
           $allNull = FALSE;
         }
         else {
+          if (!$serializeArrays && is_array($pValue) && !empty($value['serialize'])) {
+            Civi::log('use copyParams to serialize arrays', ['civi.tag' => 'deprecated']);
```

Comment:
This log message might be more helpful if it specifies the entity and field.

Reply:
Ok - added class + name since that seems to be what is easily available